templates/core/index.html

```diff
@@ -1,1157 +1,744 @@
+           role="tabpanel"
+           id="cloud-tab"
+           aria-labelledby="cloud"
+           aria-hidden="hidden">
+        <div class="grid-row u-no-margin--bottom">
+          <div class="grid-col-2 grid-col-medium-1">
+            {{ image(url="https://assets.ubuntu.com/v1/134756c3-ibm-cloud-testimonial-logo.png",
+                        alt="IBM",
+                        width="190",
+                        height="58",
+                        hi_def=True,
+                        loading="lazy",) | safe
             }}
-            <h4 class="p-heading-icon__title">研讨会</h4>
+          </div>
+          <div class="grid-col-6 grid-col-medium-3 u-no-margin--top">
+            <div class="grid-row">
+              <div class="grid-col-4 grid-col-medium-4">
+                <h3 class="p-heading--2">IBM Cloud 提供高度隔离且性能卓越的裸机服务器</h3>
+              </div>
+              <div class="grid-col-2 grid-col-medium-2">
+                <p>IBM Cloud 的裸机服务器通过使用 Ubuntu Core 提供高度隔离且高性能的租户云隔离。</p>
+                <hr class="p-rule" />
+                <p>
+                  <a href="https://canonical.com/blog/cloud-isolation-ibm-bare-metal-ubuntu-core">阅读新闻稿 &rsaquo;</a>
+                </p>
+              </div>
+            </div>
+            <div class="p-image-container">
+              {{ image(url="https://assets.ubuntu.com/v1/534be2a2-ibm-leadspace.jpg",
+                            alt="",
+                            width="1585",
+                            height="633",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
           </div>
         </div>
-        <h3 class="p-heading--4">
-          <a href="/engage/ubuntu-tech-live-episode-4"
-             onclick='dataLayer.push({ event: "GAEvent", eventCategory: "product page", eventAction: "core-center", eventLabel: "Webinar - rule4 security Ubuntu Core", eventValue: 1 });'>Ubuntu Core的网络安全分析&nbsp;&rsaquo;</a>
-        </h3>
-        <!-- rtp section center end -->
       </div>
     </div>
   </section>
 
-  <section class="p-strip--square-suru is-slanted--top-right">
-    <div class="row">
-      <div class="col-12">
-        <h2>通过SMART START快速进入市场</h2>
-        <p>
-          <a class="p-button--positive" href="https://ubuntu.com/core/smartstart">发布您的智能产品</a>
-        </p>
+    {{ vf_basic_section(
+    title={"text": "借助硬件伙伴生态加速上市"},
+    items=[
+      {
+        "type": "linked-logo-block",
+        "item": {
+          "links": [
+            {
+              "href": "https://ubuntu.com/certified?q=adlink&limit=20",
+              "text": "浏览 ADlink 的认证硬件 ›",
+              "label": "Adlink",
+              "image_attrs": {
+                "src": "https://assets.ubuntu.com/v1/6eff0263-adlink-logo.png",
+                "alt": "Adlink",
+                "width": "506",
+                "height": "481"
+              }
```

Comment:
if you use `image_html` attribute instead, you can use the `image` helper function for a better performance